awx/sso/conf.py

```diff
@@ -148,6 +148,16 @@ def __call__(self):
     placeholder=['username', 'email'],
 )
 
+register(
+    'SOCIAL_AUTH_USERNAME_IS_FULL_EMAIL',
+    field_class=fields.BooleanField,
+    default=False,
+    label=_('Use E-Mail address for usernames'),
```

Comment:
small nitpick, can we change
E-Mail to Email (here and elsewhere)? It will be more consistent with the usage in the rest of the codebase

Reply:
done